resources/metadata/cnbBuild.yaml

```diff
@@ -51,6 +51,15 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: alias
```

Comment:
would it make sense to rename to `imageAlias`?

Reply:
For what else could it possibly be an alias?

Reply:
We will rename it to `containerImageAlias` in order to be consistent with `containerImageName`, `containerImageTag` and `containerRegistryURL`